packages/core/src/index.ts

```diff
@@ -7,6 +7,7 @@ export type {
   Decoder,
   DecodedMessage,
 } from "./lib/message/version_0.js";
+export { TopicOnlyDecoder } from "./lib/message/topic_only_message.js";
```

Comment:
Pretty sure `TopicOnlyDecoder` is only used in Waku Relay so it can be moved there.